web/src/beta/components/fields/common/TextInput/index.tsx

```diff
@@ -87,4 +91,5 @@ const StyledInput = styled.input`
   :focus {
     border-color: ${({ theme }) => theme.outline.main};
   }
+  color-scheme: dark;
```

Comment:
We have never used color-scheme before, so is there a reason you are using it here instead of relying on the background and color using our theme? 🤔

Reply:
yes , as you noticed I used the default html calendar and clock and in order to re-style it I add this attribute so it will have these colors  , and this is the only way to do that or we need to work on all -webkit classes( which I tried to find all of them and I needed to add a whole css file inorder to change them and even though the style was broken) , but this way only set the scheme to be dark and then all what I need is done.

Reply:
Ok, I guess as a temp solution this is fine 🙏🏻